addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -169,10 +171,21 @@ export class DescriptionScreen extends Component {
     };
     setup() {
         this.industrySelection = useRef('industrySelection');
+        this.purposeSelectionRef = useRef("purposeSelection");
         this.state = useStore();
         this.orm = useService('orm');
+        useAutofocus();
 
         onMounted(() => this.onMounted());
+
+        onPatched(() => {
+            // Focus the next field once the previous one is done.
+            if (this.state.selectedIndustry) {
+                this.purposeSelectionRef.el.focus();
+            } else if (this.state.selectedType) {
+                this.industrySelection.el.querySelector("input").focus();
+            }
+        });
```

Comment:
Is that the right way to do this? Is not `onPatched` called on *any* change in the template? Reading this I feel like the focus will jump for no reason if we were to re-render for any other reason that filling a field in the future. Should not this be an `useEffect` based on the right state values as dependencies?